src/adapter/core.js

```diff
@@ -95,6 +97,22 @@ extend(
 
 					let editable = editor.editable();
 
+					let extraCommands = this.get('extraCommands');
+
+					const extraCommandKeys = Object.keys(extraCommands);
+					for (let i = 0; i < extraCommandKeys.length; i++) {
+						const commandName = extraCommandKeys[i];
+
+						if (editor.commands[commandName]) {
+							continue;
+						}
+
+						editor.addCommand(
+							commandName,
+							extraCommands[commandName]
+						);
+					}
+
```

Comment:
Could also use `Object.entries` here:

```javascript
Object.entries(extraCommands).forEach(([commandName, command]) => {
  if (!editor.commands[commandName]) {
    editor.addCommand(commandName, command);
  }
});
```

Reply:
I always forget about IE11 

I'll switch back to `Object.keys` because I was looking at the saucelabs logs I see:

```
IE 11.0.0 (Windows 10.0.0) ButtonBold "before all" hook FAILED
	Object doesn't support property or method 'entries' (:121)
```